generated/nidaqmx/nidaqmx_service.h

```diff
@@ -435,6 +435,8 @@ class NiDAQmxService final : public NiDAQmx::WithCallbackMethod_RegisterSignalEv
   NiDAQmxLibraryInterface* library_;
   ResourceRepositorySharedPtr session_repository_;
   ::grpc::Status ConvertApiErrorStatusForTaskHandle(::grpc::ServerContext* context, int32_t status, TaskHandle task);
+  ::grpc::Status ConvertApiErrorStatusWithReadParametersForTaskHandle(::grpc::ServerContext* context, int32_t status, int32_t samps_per_chan_read, TaskHandle task);
+  ::grpc::Status ConvertApiErrorStatusWithWriteParametersForTaskHandle(::grpc::ServerContext* context, int32_t status, int32_t samps_per_chan_written, TaskHandle task);
```

Comment:
"WithReadParameters" implies that there are multiple read parameters, but there is only one.

[style, subjective] After seeing this in context, I think it makes more sense to tack the read parameter (singular) on the end, because it is extra:
```
::grpc::Status ConvertApiErrorStatusForTaskHandleWithSampsPerChanRead(::grpc::ServerContext* context, int32_t status, TaskHandle task, int32_t samps_per_chan_read);
```

Reply:
Removed these as per the last suggestion